kibana/README.md

```diff
@@ -24,7 +24,8 @@ This helm chart is a lightweight way to configure and run our official [Kibana d
 
 | Parameter                 | Description                                                                                                                                                                                                                                        | Default                                                                                                                   |
 | ------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------- |
-| `elasticsearchURL`        | The URL used to connect to Elasticsearch.                                                                                                                                                                                                          | `http://elasticsearch-master:9200`                                                                                        |
+| `elasticsearchURL`        | The URL used to connect to Elasticsearch. Supported previous to Kibana 6.6.0.                                                                                                                                                                      | `http://elasticsearch-master:9200`                                                                                        |
```

Comment:
It's still supported but only deprecated for < 7. This makes it sound like it won't work at all in 6.6.0 which is not the case.

Reply:
👍  Pushed https://github.com/elastic/helm-charts/pull/67/commits/2c24b8c62a8d4e9cf30e0defda3c9ad8aeaa9167